package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@manifoldco/gql-zero",
   "version": "0.1.0",
   "description": "A lightweight alternative for consuming GraphQL. Zero caching, zero bells. Zero Bloat.",
-  "main": "index.js",
```

Comment:
My brain is telling me we're supposed to have this field, but maybe I'm wrong because I'm pretty sure that's not even the right path. Can you get me up to speed on why a person should or shouldn't use this field?

Reply:
We do need this but pikapkg handles it when publishing to npm. This is wrong, anyway—it should be `dist-node/index.js`. IMO we don’t need to maintain keys in package.json that pikapkg will manage.